### PR TITLE
Email verification

### PR DESCRIPTION
**Pull request by soachishti**
#50

Note:  settings.py file is updated in this request.

### CODE REVIEW

File: AutoGr/settings-sample.py [hunk 1]
```diff
@@ -126,10 +127,6 @@
 LOGIN_REDIRECT_URL = 'home'
 
 RUN_API_URL = "http://127.0.0.1:8000/autograde/api/"
-EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```
Comment (recluze) [inline AutoGr/settings-sample.py:129]: `EMAIL_BACKEND` removed?

Comment (soachishti) [reply AutoGr/settings-sample.py:129]: It was a duplicate entry. `EMAIL_BACKEND` is also defined on line 200.

Comment (recluze) [reply AutoGr/settings-sample.py:129]: Hnn, ok. Please go ahead and merge. 

Thanks.

Comment (soachishti) [reply AutoGr/settings-sample.py:129]: `AUTHENTICATION_BACKENDS` is the actual thing which is added in settings.py file (in the last line) but for some reason it isn't shown here.

https://github.com/BilalZaib/AutoGrader/blob/3dc4390591ea147696ba172bd04ea6ec8a49fb32/AutoGr/settings-sample.py

Comment (soachishti) [reply AutoGr/settings-sample.py:129]: This allows inactive user to login.

Comment (recluze) [reply AutoGr/settings-sample.py:129]: It shows in the commit (just not in this comment page).

Comment (soachishti) [reply AutoGr/settings-sample.py:129]: Okay 👍

**Pull request merged**